src/Lumi/Components/Color.purs

```diff
@@ -9,12 +9,15 @@ module Lumi.Components.Color
 import Color (rgb, rgba)
 import Color as C
 import Data.Newtype (class Newtype)
+import Foreign.Generic (class Decode, class Encode)
 
 type Color = C.Color
 
 newtype ColorName = ColorName String
 
 derive instance newtypeColorName :: Newtype ColorName _
+derive newtype instance decodeColorName :: Decode ColorName
+derive newtype instance encodeColorName :: Encode ColorName
```

Comment:
Do we really need these instances and `purescript-foreign-generic` as a dependency?

Reply:
Oh no not anymore, legacy from some experimenting I was doing. Will remove, thanks!